numpy/core/_add_newdocs.py

```diff
@@ -3751,7 +3751,7 @@
     """
     a.squeeze(axis=None)
 
-    Remove single-dimensional entries from the shape of `a`.
+    Remove entries of length one from the shape of `a`.
```

Comment:
I wonder if this would be clearer as either:
```suggestion
    Remove entries of value one from the shape of `a`.
```
or perhaps
```suggestion
    Remove axes of length one from `a`.
```

as written, this sort of implies `(n for n in a.shape if len(n) != 1)` when really it means `(n for n in a.shape if n != 1)`

Reply:
After consideration with the team we though the axes suggestion was the better approach, it's clearer and gets the point across. @ruppysuppy can you accept those changes? Thanks